app/code/core/Mage/Adminhtml/controllers/Cms/Wysiwyg/ImagesController.php

```diff
@@ -144,7 +144,7 @@ public function uploadAction()
             $targetPath = $this->getStorage()->getSession()->getCurrentPath();
             $result = $this->getStorage()->uploadFile($targetPath, $this->getRequest()->getParam('type'));
         } catch (Exception $e) {
-            $result = ['error' => $e->getMessage(), 'errorcode' => $e->getCode()];
+            $result = ['error' => true, 'message' => $e->getMessage()];
```

Comment:
This seems like a bug fix and should be in a separate PR.

Reply:
Yes the original PR contains some minor bugfixes and addition for dev mode.

Reply:
I've removed this change to keep the PR clean